domain/src/main/java/com/aconno/sensorics/domain/interactor/ifttt/publish/DeleteAnyPublishUseCase.kt

```diff
@@ -11,7 +11,7 @@ open class DeleteAnyPublishUseCase(
         return Completable.fromCallable {
             getRepositoryForPublishUseCase.execute(parameter).map { repository ->
                 repository.deletePublish(parameter)
-            }
+            }.subscribe()
```

Comment:
This doesn't look good. Is this even compilable? The return value is `Completable` and `.subscribe()` will return `Disposable`.

Reply:
It is compilable and it works. This subscribe method is called inside of Completable.fromCallable method.

Reply:
Aha OK, I missed that. But why do you need to wrap everything in a new `Completable`? Why not just return the value from a `map`?

Reply:
`map` method returns `Single<Unit>` which does not make much sense, so that is probably the reason why it has been wrapped in a new `Completable`, I just added call to subscribe method to make it work.

Reply:
You need to convert `Single<Unit>` to `Completable`

Reply:
`Single.toCompletable` is deprecated.